src/pages/signin/SignInPageLayout/Footer.js

```diff
@@ -174,7 +192,7 @@ const Footer = (props) => {
                                                 <TextLink
                                                     style={[styles.footerRow, hovered ? styles.textBlue : {}]}
                                                     href={row.link}
-                                                    onPress={row.onPress}
+                                                    onPress={row.onPress ? () => row.onPress(props.scrollViewRef) : undefined}
```

Comment:
I'm sorry, I don't understand this change. Why are we doing `onPress=undefined`

Reply:
@allroundexperts I think this is unnecessary. 
We can just do `onPress={row.onPress}`

In the callback function, we're already doing a null check for scrollViewRef